### ISSUE TITLE
CMD cannot be sized

### ISSUE THREAD

**Issue by fionafibration** [cry0h/Teensyterpreter#11]
When I run the program, CMD poos up in the dark blue text, but at the top says it cannot be resized to the rows and columns specified.

**Comment by KernelEquinox** [comment]
Yeah, this is a known issue with Windows 8/8.1/10 systems. I just pushed the new code to the dev branch, which should correct the issue. I haven't had a chance to test it though, so if you can't get the dev branch code to work, just replace line `96` in the master branch with:

``` c
Keyboard.print("cmd.exe /T:01 /K MODE CON: COLS=17 LINES=1");
```

**Comment by fionafibration** [comment]
Thanks, that helps.

**Comment by fionafibration** [comment]
You should also change the read me in the dev branch to match the new way
of inputting information.

**Comment by fionafibration** [comment]
In the new code, you don't call the parseNet() function. You just define it.

**Comment by fionafibration** [comment]
If you call it before hurryup, it works. Ignore my thing about changing the
parseNet to work. It works with one modification, and that is too call the
parseNet function before hurryup. It is better before hurryup, as it
doesn't delay anything. If you called it after, then it would put a very
slight delay in as it needs to calculate the hex. It works with that one
modification. You can close all my pull requests, they were crap. But
please merge the dev branch with the master.

**Comment by fionafibration** [comment]
That was the worst most blabbery email I have ever composed. I am sorry you
had to read that.
